eslint.config.mjs

```diff
@@ -0,0 +1,66 @@
+import { fixupConfigRules, fixupPluginRules } from '@eslint/compat';
+import { FlatCompat } from '@eslint/eslintrc';
+import importPlugin from 'eslint-plugin-import';
+
+const compat = new FlatCompat({
+  baseDirectory: import.meta.dirname,
+});
+
+export default [
+  {
+    // # .eslintignore
+    ignores: [
+      // # NOTE:
+      // # The following directives are only relevant when linting the whole
+      // # project directory, ie. running `eslint .` ⚠️
+      // # If you compile JavaScript into some output folder, exclude it here
+
+      // Generated files or folders
+      '**/node_modules/**',
+      '**/dist/**',
+      '**/build/**',
+      '**/.nx/**',
+      '**/.yarn/**',
+      // Ignore the playwright-report directory
+      'playwright-report',
+      // Ignore vendor directory in `packages/web-twig`
+      'vendor',
+
+      // Internal output folders
+      'packages/**/types/*',
```

Comment:
is this correct? Doesn't it ignore our source types?

Reply:
I have removed these rules, because those types come from `design-tokens` a they are already ignored.